CBinding/Project/CProject.cs

```diff
@@ -68,15 +67,18 @@ public enum CProjectCommands {
 	public class CProject : Project, IDeployable
 	{
 		[ItemProperty ("Compiler", ValueType = typeof(CCompiler))]
-		private ICompiler compiler_manager;
+		ICompiler compiler_manager;
 		
 		[ItemProperty ("Language")]
-		private Language language;
+		Language language;
 		
-		[ItemProperty("Target")]
-		CBinding.CompileTarget target = CBinding.CompileTarget.Bin;
+		[ItemProperty ("OutputType")]
+		CompileTarget target = CompileTarget.Exe;
+
+		public CLangManager cLangManager;
+		public ClangProjectSymbolDatabase db;
 		
-    	private ProjectPackageCollection packages = new ProjectPackageCollection ();
```

Comment:
sorry these packagecollection and alike are legacy of previous CBinding, will remove them completely when merged into Elsayed's CMakeProject